components/quiz-card.html

```diff
@@ -0,0 +1,59 @@
+<section class="card">
+  <a class="link_bookmark" href="#bookmark"
```

Comment:
BEM naming convention:
```suggestion
  <a class="card__bookmark" href="#bookmark"
```